css/style.css

```diff
@@ -1,19 +1,169 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/*GENERAL style*/
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    width: 100%;
+}
+
```

Comment:
Make sure you follow consistency in your spacing between CSS selectors/ styling e.g between line 10 and 11 could be reduced to one gap and applied throughout your CSS code. Otherwise, I have to complement you on your amazing CSS accurate styling and selectors (pseudo-class).